### PR TITLE
Request completion entry details from auto-import entries.

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
Also doubles the auto-import request rate from 1% to 2%.

### CODE REVIEW

File: src/utils/exerciseServer.ts [hunk 1]
```diff
@@ -311,6 +314,30 @@ async function exerciseServerWorker(testDir: string, tsserverPath: string, repla
...
                         });
+
+                        // Auto-import completions frequently cause issues with completion entry details.
+                        if (requestWithAutoImports) {
+                            const completionEntries = invokedResponse.body.entries;
+                            for (let entry of completionEntries) {
+                                if (entry.hasAction && entry.source !== undefined && "data" in entry) {
+                                    const { name, source, data } = entry;
+                                    
+                                    // 'completionEntryDetails' can take multiple entries; however, it's not useful for diagnostics
```
Comment (DanielRosenwasser) [inline src/utils/exerciseServer.ts:325]: Curious to hear if others think this is bad - this is pretty likely to be slower, but I like to look at the last command and know exactly what went wrong.

I could make requests on 5 at a time in the future.

Comment (andrewbranch) [reply src/utils/exerciseServer.ts:325]: AFAIK nobody uses the multiple-details-at-a-time command. IIRC it’s not optimized whatsoever; it just loops at the super top level and doesn’t cache anything.

**Review by andrewbranch — APPROVED** [review]
Details requests are usually pretty fast; I doubt this will slow the whole process down that much.

**Review by gabritto — APPROVED** [review]

**Review by navya9singh — APPROVED** [review]

**Comment by DanielRosenwasser** [comment]
https://github.com/microsoft/TypeScript/issues/52660 is the first run with this enabled... and it is brutal!

**Pull request merged**